Console: Allow ETH to be sent through the act command (#1355)
aragon/aragon: docs/CONSOLE.md

```

The above command will select the Agent at `0x77df...b027`, interact with the Voting app at `0xe96c...e7ae` (installed on a different organization), and execute the method `vote(uint256 voteId, bool supports, bool executesIfDecided)`.

> The act command can receive the amount of ETH to send using the last parameter of the command, in wei. This parameter is optional. You can also use the exec() command with the necessary encoded calldata.

Reviewer: > You can also use the `exec()` command with the encoded calldata necessary.

Hmm, not sure I understand this, but if we're mentioning using `exec` to call `act`, I would just leave it out now since there's no good reason to do so.

-----

I saw we've already updated the first sentence a bit, but I'd also recommend:

```suggestion
As a final optional parameter, the `act` command can receive the amount of ETH, in wei, to send with the transaction.
```

Author: Ah, this is much clearer!

> Here's an example of how to send 0.5 ETH to a contract:

Reviewer: ```suggestion
Here's an example of how to send 0.5 ETH to a contract using the Agent's own balances:
```

The above command will select the Agent at `0x77df...b027`, interact with the Voting app at `0xe96c...e7ae` (installed on a different organization), and execute the method `vote(uint256 voteId, bool supports, bool executesIfDecided)`.
 

> The last parameter the `act` command can receive, which is optional, is the amount of ETH to send, in wei. You can also use the `exec()` command with the encoded calldata necessary.

Reviewer 2: Should we add an example here, to make it clear that it is a command parameter rather than a method parameter?

Also I was wondering if that sentence could be made a little bit clearer:

>  The `act` command can receive the amount of ETH to send using the last parameter of the command, in wei. This parameter is optional. You can also use the `exec()` command with the necessary encoded calldata.

Author: Yep, let's add an example. 👍

Author: Done in [75b083e](https://github.com/aragon/aragon/pull/1355/commits/75b083e3a4adf26fc401e3c79bee1980a505e85c)
